recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 build:
   number: 1
```

Comment:
Wait a min... Don't we need to bump build number?

Reply:
Nope. The version+revision never existed for windows so this is additive. I don't see a reason to deploy another revision for the other platform/arch combos because the program hasn't changed.

Reply:
How do we check that it is really there now?

Reply:
If you want me to rebuild with a different revision I can. The 0.01% of users unpacking the Linux or Mac package and building the bundled recipe locally on Windows will be sad. Otherwise the slight mismatch in the metadata affects no one.

Reply:
nvm looks like it works on win64 now. Sorry for the noise!